components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-default.component.html

```diff
@@ -9,7 +9,11 @@
         <app-form-image
           [parentForm]="formCtrl"
           [images]="config?.image?.options"
-          [readonly]="config?.image?.readOnly"
+          [jupyterReadonly]="config?.image?.readOnly"
+          [imagesVSCode]="config?.imageVSCode?.options"
```

Comment:
These will need some slight modification to look at, like `config?.image?.vscode` instead

Reply:
Not just these, but also how the types.ts and config.ts work if I'm correct (not looking at the code at the moment).